libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionsVcEvidence.java

```diff
@@ -35,13 +43,29 @@ public AuditExtensionsVcEvidence(
             @JsonProperty(value = "iss", required = false) String iss,
             @JsonProperty(value = "evidence", required = false) String evidence,
             @JsonProperty(value = "successful", required = false) Boolean successful,
+            @JsonProperty(value = "vot", required = false) Vot vot,
             @JsonProperty(value = "isUkIssued", required = false) Boolean isUkIssued,
             @JsonProperty(value = "age", required = false) Integer age)
-            throws JsonProcessingException {
-        this.iss = iss;
-        this.evidence = evidence == null ? null : new ObjectMapper().readTree(evidence);
-        this.successful = successful;
-        this.isUkIssued = isUkIssued;
-        this.age = age;
+            throws AuditExtensionException {
+        try {
+            this.iss = iss;
+            this.evidence = evidence == null ? null : OBJECT_MAPPER.readTree(evidence);
+            this.successful = successful;
+            this.vot = vot;
+            this.isUkIssued = isUkIssued;
+            this.age = age;
+        } catch (JsonProcessingException e) {
+            throw new AuditExtensionException(e.getMessage());
```

Comment:
Not sure, would it be more complete to pass the entire error?

`AuditExtensionException(e);`

Reply:
That's a good question! But the evidence block of the VC shouldn't contain information about the subject. That's in the credentialSubject section.